Sources/Core/Filters/FreeText/FreeTextFilterViewController.swift

```diff
@@ -5,26 +5,35 @@
 import FinniversKit
 import UIKit
 
-public protocol SearchQuerySuggestionsDataSource: AnyObject {
-    func searchQueryViewController(_ searchQueryViewController: SearchQueryViewController, didRequestSuggestionsFor searchQuery: String, completion: @escaping ((_ text: String, _ suggestions: [String]) -> Void))
+public protocol FreeTextFilterDataSource: class {
+    func freeTextFilterTableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int
```

Comment:
Sending the tableView with the delegate exposes unnecessary implementation detail to the delegate. Delegate should not need/care about what kind of view is used to show it? It also makes it difficult to change to another view type to show suggestions in.

Reply:
I tried to keep the FreeTextViewController internal, but I guess it makes sense to make it public when it is going to call the main app anyways. I'll make it a bit more custom ;)

Reply:
You could make a small protocol that is public maybe, all you need is a method to tell that suggestions are ready, right? Or go back to the completion handler that was there. 

I think you might need to have the connection between search text and suggestions anyway... You might have timing issues otherwise, i.e. suggestions for "iPh" might take longer to come back than suggestions for "iPho". Or queuing it and cancelling old request.

Reply:
But actually, rather remove all the suggestion things for now then spend a lot of time on it. It is not needed for initial release.

Reply:
Allright :)